### DISCUSSION TITLE
Welcome to Vim9 LSP plugin discussions!

### DISCUSSION THREAD

**Discussion by yegappan** [yegappan/lsp#458] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ubaldot** [comment]
Awesome! I sometimes have questions which are not really issues or features requests, and this space looks very spot on! 

As requested: I am a control system engineer and this plugin definitely makes me to stay with Vim. 

Question: LSP looks very mature. What about including it in the standard Vim distribution? :)

**Comment by D029-W** [comment]
Great plugin! The document was maintained very well.

And, it doesn't add surprise key-binds to my keyboard. I really appreciate this.

**Comment by darkseid-is** [comment]
Thanks for a great plugin. The following features will be nice to have 

1. Pass custom flags to lsp server , for example rust-analyzer supports FormatOnSave.

2. Lsp Progress as mentioned in issue [https://github.com/yegappan/lsp/issues/453](url), rust-analyzer initial check is heavy resource intensive for large projects. so it would be nice to have a progress bar to notify when the lsp has finished checking. 

3. Stop a running LspServer.

**Comment by damonmccain** [comment]
Liking this plugin, I am learning vim and vimscript and would like to contribute to a plugin like this in the near future. Would that be possible?

**Comment by jclsn** [comment]
Hi, I am really looking forward to seeing this plugin mature! 

I'd love to switch to a native LSP plugin! The responsiveness is great! I've been checking every few weeks for updates and am always happy to see progress. The main thing that is keeping me from using it is language server concurrency to e.g. use clangd and cpell-lsp on the same file.
